Silicon/RaptorlakePkg/Rplps/Fsp/FspBinRplps.inf

```diff
@@ -0,0 +1,27 @@
+## @file
+#  File to describe FSP repo information
+#
+#  Copyright (c) 2020 - 2024, Intel Corporation. All rights reserved.<BR>
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#
+##
+
+
+[UserExtensions.SBL."CloneRepo"]
+  REPO    = https://github.com/intel/FSP.git
+  COMMIT  = 0efd8a3723568012e8ac2ccc86f6beef1b440230
+
+[UserExtensions.SBL."CopyList"]
+# For RPL-P
```

Comment:
@bejeanmo fix the board name, it's "RPL-PS"

Reply:
Good catch Kevin. Should be fixed.